File: HomeBudgetCalculator.Infrastructure/DTO/IncomeDTO.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace HomeBudgetCalculator.Infrastructure.DTO
{
    public class IncomeDTO
    {
        public Guid Id { get; set; }

        public string Title { get; set; }

        public decimal Value { get; set; }

        public DateTime Date { get; set; }
    }
}

File: Frontend/src/app/DTO/IncomeDTO.ts
export class IncomeDTO {
  constructor(budgetId: string, title: string, value: number, date: Date) {
    this.id = budgetId;
    this.title = title;
    this.value = value;
    this.date = date;
  }

  id: string
  title: string
  value: number
  date: Date
}
